lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Helpers/FB_MirrorTwoCoatingProtection.TcPOU

```diff
@@ -96,27 +109,57 @@ IF bUsePmpsDb THEN
     END_IF
 
     IF bBPsLoaded THEN
-        nUpperCoatingBitmask := aDbStateParams[0].stBeamParams.neVRange;
-        nLowerCoatingBitMask := aDbStateParams[1].stBeamParams.neVRange;
+        E_CoatingPos := Upper;
```

Comment:
I'll keep this in mind for future use, I'm not 100% how this compares to making a real enum given that we have to swap it around like this. Mixed feelings for sure, but the code is very clear in its intent now.

Reply:
Yea, I thought this was weird. I couldn't use it how I thought I should be able to. That is, `E_CoatingPos.Upper` wouldn't work.